### PR TITLE
Give more help for login problems

### PR DESCRIPTION
**Pull request by urielsalis**
Make it one of the first choices, so users dont miss it
Ask if they have resetted their password and link them to the hacked article

### CODE REVIEW

**Review by tyteen4a03 — APPROVED** [review]

**Comment by urielsalis** [comment]
+1 to the changes, merge when you are ready

File: mch-questions.yml [hunk 1]
```diff
@@ -64,6 +75,9 @@ questions:
       [this Mojang Support article](https://help.mojang.com/customer/portal/articles/361483) for information on how to
       get your account back.
 
+      Note that our volunteers **have no access to account information**, and can't help you with account issues.
+      If you already sent a email to Mojang, please wait for their response.
```
Comment (LordRalex) [inline mch-questions.yml]: We should link the form to email Mojang as well here, to push it toward users more.

File: mch-questions.yml [hunk 2]
```diff
@@ -45,9 +47,18 @@ questions:
...
-      to: account-issues.unable-log-in.is-throttled
+      to: account-issues.unable-log-in.reset-password
     - text: 'No'
       to: technical-issues.ready
+  account-issues.unable-log-in.reset-password:
+    heading: ''
+    description: ''
+    question: Have you tried [resetting your password](https://account.mojang.com/password)? Make sure to check the spam folder. The reset password email may take up to 15 minutes to arrive.
+    choices:
+    - text: 'Yes'
```
Comment (LordRalex) [inline mch-questions.yml]: This doesn't seem like a good place for a "Yes" to me. Did you try -> several scenarios can occur here, which a "Yes" doesn't really work out well. "I changed my password" is probably more accurate.

File: mch-questions.yml (hunk 1 above)
Comment (urielsalis) [reply mch-questions.yml]: Done, I made "sent a email to mojang" a link

File: mch-questions.yml (hunk 2 above)
Comment (urielsalis) [reply mch-questions.yml]: Changed it to Yes, and the new password doesnt work

**Pull request merged**